# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../wmint/prefix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

...

  - Rene Rebe <rene@exactcode.de>

--- wmint/Makefile	2000-01-08 21:25:18.000000000 +0100
+++ wmint/Makefile	2005-05-30 16:16:26.000000000 +0200
@@ -20,7 +20,7 @@
 	rm wmint
 
 install:: wmint
-	cp -f wmint /usr/local/bin/
-	chmod 755 /usr/local/bin/wmint
-	chown root:root /usr/local/bin/wmint
+	cp -f wmint /usr/bin/
+	chmod 755 /usr/bin/wmint
+	chown root:root /usr/bin/wmint
 	@echo "wmint installation finished..."
